layouts/blog/single.html

```diff
@@ -28,6 +28,21 @@
             <div class="disqus">
                 {{ template "_internal/disqus.html" . }}
             </div>
+            {{ if .Site.Params.gitalk }}
+            <div id="gitalk-container"></div>
+              <script type="text/javascript">
```

Comment:
minor: the script element should be indented one step towards the left

Reply:
Yeah I will modify it along with the config.toml. Am I just going to create another pull request?

Reply:
you can push additional commits to the same `dev/gitalk` branch, and the commits will appear in this pull request.